monkey/infection_monkey/post_breach/actions/modify_shell_startup_files.py

```diff
@@ -0,0 +1,32 @@
+from common.data.post_breach_consts import POST_BREACH_SHELL_STARTUP_FILE_MODIFICATION
+from infection_monkey.post_breach.pba import PBA
+from infection_monkey.utils.shell_startup_files_modification import\
+    get_commands_to_modify_shell_startup_files
+from infection_monkey.utils.environment import is_windows_os
+
+
+class ModifyShellStartupFiles(PBA):
+    """
+    This PBA attempts to modify shell startup files,
+    i.e. ~/.bashrc, ~/.profile, ~/.bash_profile in linux,
+    and profile.ps1 in windows.
+    """
+
+    def __init__(self):
+        super(ModifyShellStartupFiles, self).__init__(name=POST_BREACH_SHELL_STARTUP_FILE_MODIFICATION)
```

Comment:
you can just do `super()` in python3.7

Reply:
Create a class called `ModifyShellStartupFile(PBA)` in `monkey/infection_monkey/post_breach/shell_startup_files`. This class is a simple container for a single post breach action. It contains windows and linux command, but executes only one of them based on OS.
Create a class called `ShellStartupPBAgenerator` - use this class to generate `ModifyShellStartupFile` objects from commands. 
So, the workflow should be as follows: 
1. `ModifyShellStartupFiles` calls `ShellStartupPBAgenerator.get_modify_shell_startup_pbas() -> List[ModifyShellStartupFile]`. 
2. `ShellStartupPBAgenerator` class builds commands and returns a list of `ModifyShellStartupFile` PBA objects.
3. `ModifyShellStartupFiles` stores the list of PBA's. On `run()` it simply does `[pba.run() for pba in self.modify_shell_startup_PBA_list]`.

Reply:
Okay!
Almost done with this; I'll push all the changes together tomorrow.

Reply:
@VakarisZ, just pushed! 

> you can just do super() in python3.7

I left this as it is, for now, to maintain consistency since that's how it has been called everywhere else.